tests/test_main.py

```diff
@@ -0,0 +1,40 @@
+import unittest
+from main import parse_data
+from models import Marker
+
+class TestParseData(unittest.TestCase):
+
+  def setUp(self):
+      self.marker_dummy = dict(type=1, title="test title", description="test description", latitude=1, longitude=1)
+      self.bad_marker_dummy = dict(type=1, title="No properties")
+
+  def tearDown(self):
+    self.marker_dummy = None
+    self.bad_marker_dummy = None
+
+  class NoParsedObject(object):
+    """docstring for ClassName"""
+    def __init__(self, arg):
+      self.arg = arg
+
+  def test_class_null(self): 
+    self.assertEqual(parse_data(None, self.marker_dummy), None)
+
+  def test_data_null(self):
+    self.assertEqual(parse_data(Marker, None), None)
+
+  def test_bad_data_format_marker_class(self):
+    self.assertEqual(parse_data(Marker, self.bad_marker_dummy), None)
+
+  def test_class_without_parse_method(self):
+    self.assertEqual(parse_data(self.NoParsedObject, self.marker_dummy), None)
+
+  def test_pares_marker(self):
```

Comment:
Good eye @omerxx, I've fix that misspelling, thanks.

I've used the standard python unit testing framework, unittest. If you like there is more info in the [docs](https://docs.python.org/2/library/unittest.html).